installer/Migration/V5_1_0/permission/api.yaml

```
@@ -128,32 +128,33 @@ apiv2_performance_performance_tracker_logs:
     - { role: Admin, permission: { read: true, create: true, update: true, delete: true, self: false } }
     - { role: ESS, permission: { read: true, create: true, update: true, delete: true, self: true } }
 
-apiv2_performance_tracker_reviewers_autocomplete:
-  description: 'API-v2 Performance - Performance Tracker Reviewers Autocomplete'
-  api: OrangeHRM\Performance\Api\TrackerReviewerAPI
+apiv2_performance_review_kpi_list:
+  description: 'API-v2 Performance - Performance Review Kpi List'
+  api: OrangeHRM\Performance\Api\ReviewKpiAPI
   module: performance
   allowed:
     read: true
     create: false
     update: false
     delete: false
   permissions:
-    - { role: Admin, permission: { read: true, create: false, update: false, delete: false, self: true } }
-    - { role: Reviewer, permission: { read: true, create: false, update: false, delete: false, self: true } }
+    - { role: Admin, permission: { read: true,  create: false, update: false, delete: false } }
+    - { role: Supervisor, permission: { read: true,  create: false, update: false, delete: false } }
+    - { role: ESS, permission: { read: true,  create: false, update: false, delete: false } }
 
-apiv2_performance_review_kpi_list:
-  description: 'API-v2 Performance - Performance Review Kpi List'
-  api: OrangeHRM\Performance\Api\ReviewKpiAPI
+apiv2_performance_review_allowed_actions:
+  description: 'API-v2 Performance - Performance Review Allowed Actions'
+  api: OrangeHRM\Performance\Api\PerformanceReviewAllowedActionsAPI
   module: performance
   allowed:
     read: true
     create: false
     update: false
     delete: false
   permissions:
-    - { role: Admin, permission: { read: true,  create: false, update: false, delete: false } }
-    - { role: Supervisor, permission: { read: true,  create: false, update: false, delete: false } }
-    - { role: ESS, permission: { read: true,  create: false, update: false, delete: false } }
+    - { role: Admin, permission: { read: true, create: false, update: false, delete: false, self: false } }
+    - { role: Supervisor, permission: { read: true, create: false, update: false, delete: false, self: false } }
+    - { role: ESS, permission: { read: true, create: false, update: false, delete: false, self: false } }
```

```suggestion
    - { role: Admin, permission: { read: true, create: false, update: false, delete: false } }
    - { role: Supervisor, permission: { read: true, create: false, update: false, delete: false } }
    - { role: ESS, permission: { read: true, create: false, update: false, delete: false } }
```
@devishke-orange @SharukaNP @SandeepaOHRM 
will remove self from all APIs, default is false. Also no usage of self in API. but don't remove from other data groups. remove only for API data groups with prefix `apiv2_`